### ISSUE TITLE
Does not work in product detail view anymore

### ISSUE THREAD

**Issue by crysix** [Vinai/nicer-image-names#28]
At least in magento 1.9.3+ (maybe also earlier versions) the extension only working in product list, but not in detail view anymore.
In the product details the image name is only ".jpg" the complete name part is missing.
In the list view it is working normally.

**Comment by magentix98** [comment]
We are on 1.9.2.2. 
List view is ok. In product detail we get just the normal unaltered filename.
Are there any ideas for a solution already?

**Comment by crysix** [comment]
In 1.9.2.3, I only see ".jpg" as filename. I found out, that this only happens to the product detail image. All thumbs are generated correctly. No idea yet and no time to look at this problem right now.

**Comment by magentix98** [comment]
ok, i have solved my problem. it was a rewrite conflict. your problem seems to be different, as you get ab empty file name.
Could you debug into Netzarbeiter_NicerImageNames_Model_Image::setBaseFile ? what happens to $filename in there?

**Comment by crysix** [comment]
Yes, it's not a rewrite conflict, it happens after upgrate to Magento 1.9.2.3., in 1.9.2.2 it worked fine (or at least I think it does). I have no time at the moment to debug.
